src/Poker/Holdem.hs

```diff
@@ -0,0 +1,52 @@
+
+-- | ...
+-- Module      : Poker.Holdem
+-- Description : Texas Hold'em module
+-- Copyright   : (c) Ghais Issa 2021
+--
+-- A variant of poker where two cards, known as hole cards, are dealt face down to each player,
+-- and then five community cards are dealt face up in three stages. The stages consist of a series
+-- of three cards ("the flop"), later an additional single card ("the turn" or "fourth street"),
+-- and a final card ("the river" or "fifth street"). Each player seeks the best five card poker hand
+-- from any combination of the seven cards; the five community cards and their two hole cards.
+-- ...
+module Poker.Holdem
+  (
+    Hole(..)
+  , Flop(..)
+  , Turn(..)
+  , Street(..)
+  , Community(..)
+  , Hand(..)
+  , randomHand
+  ) where
+
+import           Data.Random.RVar (RVar)
+import           Poker.Deck (Card, draw1_, shuffle, stdDeck)
+
+-- | Player's 2 cards.
+data Hole = Hole !Card !Card deriving stock (Show, Eq)
```

Comment:
I like the word Hole for this data structure. I always called it Hand (which was always confusing) and I could never think of anything ebtter...